static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+            }
+          ],
+          "parameters": [
+            {
+              "in": "query",
+              "name": "offset",
+              "required": false,
+              "schema": {
+                "type": "integer",
+                "minimum": 0,
+                "default": 0
+              },
+              "description": "The number of items to skip before starting to collect the result set."
+            },
+            {
+              "in": "query",
+              "name": "limit",
+              "required": false,
+              "schema": {
+                "type": "integer",
+                "minimum": 1,
+                "default": 10
+              },
+              "description": "The numbers of items to return."
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "A paginated list of jobs.",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "type": "object",
+                    "properties": {
+                      "count": {
+                        "type": "integer",
+                        "description": "The total number of jobs available.",
+                        "example": 5
+                      },
+                      "result": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "key": {
+                              "type": "string",
+                              "example": "123123-131231-123123-123123"
+                            },
+                            "cronExpression": {
+                              "type": "string",
+                              "nullable": true,
+                              "example": "*/30 * * * *"
+                            },
+                            "isMobile": {
+                              "type": "boolean",
+                              "example": false
+                            },
+                            "url": {
+                              "type": "string",
+                              "example": "https://something.com/something/"
+                            },
+                            "isEnabled": {
+                              "type": "boolean",
+                              "example": true
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "400": {
+              "description": "Bad request. Possible issue with the pagination parameters."
+            },
+            "500": {
+              "description": "Internal server error."
+            }
+          }
+        },
+        "post": {
+          "tags": [
+            "PPM"
+          ],
+          "summary": "Create a PPM job",
```

Comment:
Here you say "create a ppm job".  So above, might also say get a list of ppm jobs.